src/Uno.UI/UI/Xaml/Controls/Primitives/ButtonBase/ButtonBase.Android.cs

```diff
@@ -32,12 +33,14 @@ protected override void OnUnloaded()
 			_isEnabledSubscription.Disposable = null;
 			_touchSubscription.Disposable = null;
 
-			Tapped -= HandleTapped;
+			PreRaiseTapped -= OnPreRaiseTapped;
 		}
 
-		private void HandleTapped(object sender, TappedRoutedEventArgs e)
+		private void OnPreRaiseTapped(object sender, EventArgs e)
```

Comment:
Changing Click to be called from the tapped gesture recogniser instead of PointerReleased is a rather big change, which will certainly lead to subtle regressions. Is it necessary? Also, is it correct? The fact that PointerReleased is never raised for Button on UWP, suggests that 'under the hood' the `Click` is coming from `PointerReleased` timing-wise.

cc @dr1rrb

Reply:
Events fired when double-clicking a `Button` on UWP:
```
EventTestButton_Click, time=2019-06-05 12:33:02 PM -04:00 - 151
EventTestButton_Tapped, time=2019-06-05 12:33:02 PM -04:00 - 155
EventTestButton_DoubleTapped, time=2019-06-05 12:33:02 PM -04:00 - 206
EventTestButton_Click, time=2019-06-05 12:33:02 PM -04:00 - 315
```

Reply:
The _real_ fix will come from another PR.  This one is a quick fix for broken flows.